nursery/reference-doh.yml

```diff
@@ -0,0 +1,75 @@
+rule:
+  meta:
+    name: reference of DoH
+    namespace: communication/dns
+    author: markus.neis@swisscom.com
+    scope: file
+    references:
+      - https://github.com/curl/curl/wiki/DNS-over-HTTPS
+    examples:
+      # - 749e7becf00fccc6dff324a83976dc0d
```

Comment:
can you update this line to be:
  1. not commented, and
  2. include the function offset in which the string is found?
    a. you can use `capa -v` to figure out what its recognizing
    b. format it like `749e7...:0x401000`

Reply:
and finally, would you add the example file to https://github.com/fireeye/capa-testfiles/

Reply:
> can you update this line to be:
> 
> 1. not commented, and
> 2. include the function offset in which the string is found?
>    a. you can use `capa -v` to figure out what its recognizing
>    b. format it like `749e7...:0x401000`

Done!

Reply:
> and finally, would you add the example file to https://github.com/fireeye/capa-testfiles/

Added in https://github.com/fireeye/capa-testfiles/pull/32

Reply:
Correction: had to close the previous PR and added now in https://github.com/fireeye/capa-testfiles/pull/33 adjusted filename